src/idpyoidc/server/oauth2/introspection.py

```diff
@@ -103,7 +104,7 @@ def process_request(self, request=None, release: Optional[list] = None, **kwargs
             _session_info = _context.session_manager.get_session_info_by_token(
                 request_token, grant=True
             )
-        except (UnknownToken, WrongTokenClass):
+        except (UnknownToken, WrongTokenClass, ToOld):
```

Comment:
This change is important to properly mark token with `active: false` on the introspection endpoint, when the token has expired.

Reply:
(Maybe this should be split into its own PR, along with the relevant test case.)